### PR TITLE
jyanand-fix-tests

### PR DESCRIPTION
**Pull request by Jyotsna-Anand**
## Description

Fixing failures in scenario tests. 

## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by markcowl** [comment]
@Jyotsna-Anand You will need to pull the latest and fix merge conflicts

**Comment by Jyotsna-Anand** [comment]
@markcowl I have checked the box to allow edits from maintainers. Can you please help merge the latest changes in master? Thank you!

File: src/Compute/Compute/Extension/AzureDiskEncryption/SetAzureDiskEncryptionExtension.cs [hunk 1]
```diff
@@ -541,9 +542,22 @@ public override void ExecuteCmdlet()
 
                     currentOSType = virtualMachineResponse.StorageProfile.OsDisk.OsType;
 
+                    var vmParameters = (this.ComputeClient.ComputeManagementClient.VirtualMachines.Get(
+                        this.ResourceGroupName, VMName));
+
                     if (OperatingSystemTypes.Linux.Equals(currentOSType) && !SkipVmBackup)
                     {
-                        CreateVMBackupForLinx();
+                        if (vmParameters.StorageProfile.OsDisk.ManagedDisk != null)
```
Comment (markcowl) [inline src/Compute/Compute/Extension/AzureDiskEncryption/SetAzureDiskEncryptionExtension.cs]: We need a changelog update for this specific fix.

File: src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs [hunk 2]
```diff
@@ -115,9 +115,16 @@ public void TestAzureDiskEncryptionExtensionSinglePassDisableAndRemove()
 
         [Fact]
```
Comment (markcowl) [inline src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs]: What is AzureRM.Compute.zip?  Please remove this.

Comment (Jyotsna-Anand) [reply src/Compute/Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs]: Sorry, that got added accidentally. Removed this in my last commit.

File: src/Compute/Compute/Extension/AzureDiskEncryption/SetAzureDiskEncryptionExtension.cs (hunk 1 above)
Comment (Jyotsna-Anand) [reply src/Compute/Compute/Extension/AzureDiskEncryption/SetAzureDiskEncryptionExtension.cs]: Added an entry in changelog for this fix.

**Pull request merged**